etn-ccis/blui-react-native-themes
# Add MD3 themes

Fixes BLUI-4353 & 4363.



#### Changes proposed in this Pull Request:

- Add MD3 light and dark theme



#### Screenshots / Screen Recording (if applicable)
light theme

![Simulator Screenshot - iPhone 14 - 2023-07-25 at 14 43 10](https://github.com/etn-ccis/blui-react-native-themes/assets/13989985/bc86236a-3971-42d5-8d70-95afca902521)

dark theme (background color isn't working properly right now)
![Simulator Screenshot - iPhone 14 - 2023-07-25 at 15 16 12](https://github.com/etn-ccis/blui-react-native-themes/assets/13989985/caa88751-0fb8-45b0-8a5d-06307ca8cd43)




#### To Test:

- This will be difficult to test, but I want to get these in so we can test out the showcase and components more easily moving forward. There also seems to be an issue with the background color which I'll try to resolve, but may be a task for another time as I passively working on this so I'm just committing what I have for now.
- You can test this by:
- generate a new react native project via cli `npx react-native@latest init AwesomeProject`
- Then follow the [react native paper documentation](https://callstack.github.io/react-native-paper/docs/guides/getting-started) to get set up with react native paper
- Use the theme and add some components to the screen to see them working.



#### Any specific feedback you are looking for?

- These are still a work in progress, but feel free to copy them and fix the bugs for whoever takes on the theme stories
